js/piwik.js

```diff
@@ -2855,6 +2857,11 @@ if (typeof window.Matomo !== 'object') {
                     callback({request: request, trackerUrl: configTrackerUrl, success: true, isSendBeacon: true});
                 }
 
+                // If the query parameter indicating this is a test exists, close after first request is sent
+                if (wasJsTrackingCodeInstallCheckParamProvided()) {
+                    window.close();
```

Comment:
The JSLinter tests are currently failing. Guess you need to use windowAlias here instead.